### DISCUSSION TITLE
Add customization for SuddenDeath: different fail conditions

### DISCUSSION THREAD

**Discussion by ramjam18** [ppy/osu#15012] [Ideas] [answered]
I dunno where else to post feature requests, so pardon me if this is the wrong place.

Perhaps add a few more fail conditions to choose from in Mod Customization?

We already know about the standard "**Combo Break**" condition that makes you fail when you don't hit a note or miss a slider tick/repeat.
But I also thought about a condition where you fail _if you don't correctly alternate between keys_. Sort of like a "**Used Same Key Twice In A Row**" condition.

Perhaps, if there are some other interesting ideas (like _fail if acc can no longer be above n%_, or others), maybe consider adding those too? ...I'd be thrilled to see some additions like this!

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13120

**Reply by ramjam18** [reply]
ok i'll go ahead and try to close this--

i can't
